frontend/src/app/About/Contributors.tsx

```diff
@@ -0,0 +1,96 @@
+import { Globe } from 'iconoir-react';
+import { contributorStructure } from './index';
+import styles from "./Contributors.module.scss"
+
+interface ContributorsProps {
+	currContributors: contributorStructure[];
+	alumniContributors: contributorStructure[];
+}
+
+
+const Contributors: React.FC<ContributorsProps> = ({ currContributors, alumniContributors }) => {
+	/* 	console.log(currContributors)
+		console.log(alumniContributors)
+		console.log(currContributors[0]?.name)
+		console.log(currContributors[0]?.img.seriousBase64);
+	
+	 */
+
+	console.log(currContributors)
+	const base64Root = 'data:image/jpeg;base64,'
+
+	const alumniByGradYear: { [key: number]: contributorStructure[] } = {};
+
+	// Populate the data structure
+	alumniContributors.forEach((member) => {
+		const gradYear = member.gradYr;
+		if (gradYear in alumniByGradYear) {
+			alumniByGradYear[gradYear].push(member);
+		} else {
+			alumniByGradYear[gradYear] = [member];
+		}
+	});
+
+
+
+	return (
+		<>
+			<div className={styles.currentContributors}>
+				<h1>
+					Current Contributors
+				</h1>
+				<div>
+					{currContributors.map(({ name, img, websiteURL, role }) => (
```

Comment:
It might be easier to include these loops within `About/index.tsx` and instead separate `Contributor` and `Alumni` into their own reusable components that you pass `name, img, websiteURL, etc.` into from the loop.